deepes/include/torch_predictor.h

```diff
@@ -40,7 +42,16 @@ class Predictor{
     load_proto_conf(config_path, *_config);
     _sampling_method = std::make_shared<GaussianSampling>();
     _sampling_method->load_config(*_config);
-    _optimizer = std::make_shared<SGDOptimizer>(_config->optimizer().base_lr());
+
+    std::string opt_type = _config->optimizer().type();
+    std::transform(opt_type.begin(),opt_type.end(),opt_type.begin(),::tolower);
+    if (opt_type == "sgd") {
+      _optimizer = std::make_shared<SGDOptimizer>(_config->optimizer().base_lr());
```

Comment:
Please add additional configuration term for these optimizers, like momentum for sgd and epsilon for `Adam` optimizer.

Reply:
done